stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -680,19 +706,6 @@ function getproperty(C::Cholesky{<:Any,<:Diagonal}, d::Symbol)
 end
 
 Base._sum(A::Diagonal, ::Colon) = sum(A.diag)
-function Base._sum(A::Diagonal, dims::Integer)
```

Comment:
Is this deletion related?

Reply:
No, that must have been my cat on the keyboard. I have no idea...